### PR TITLE
chore(docs): clarify that variables be edited through the web UI

### PR DESCRIPTION
**Pull request by phorcys420**
When visiting a community user 2 weeks ago, they pointed out that we don't clearly state that template-wise variables can be edited through the web UI.

This PR adds a line to (hopefully) clarify this.
I didn't find other places where this could be added, but feel free to let me know!

### CODE REVIEW

File: docs/admin/templates/extending-templates/variables.md [hunk 1]
```diff
@@ -74,9 +74,10 @@ for providing values to variables using the Coder CLI:
 
 1. _Manual input in CLI_: You can manually input values for Terraform variables
    directly in the CLI during the deployment process.
-2. _Command-line argument_: Utilize the `--var name=value` command-line argument
+2. _Web UI_: You can set and/or edit variable values under "Variables" in the template's settings.
```
Comment (EdwardAngert) [inline docs/admin/templates/extending-templates/variables.md:77]:
```suggestion
1. _Web UI_: You can set or edit variable values under **Variables** in the template's settings.   
```

File: docs/admin/templates/extending-templates/variables.md [hunk 2]
```diff
@@ -74,9 +74,10 @@ for providing values to variables using the Coder CLI:
 
 1. _Manual input in CLI_: You can manually input values for Terraform variables
    directly in the CLI during the deployment process.
-2. _Command-line argument_: Utilize the `--var name=value` command-line argument
+2. _Web UI_: You can set and/or edit variable values under "Variables" in the template's settings.   
+3. _Command-line argument_: Utilize the `--var name=value` command-line argument
```
Comment (EdwardAngert) [inline docs/admin/templates/extending-templates/variables.md:78]:
```suggestion
1. _Command-line argument_: Utilize the `--var name=value` command-line argument
```

File: docs/admin/templates/extending-templates/variables.md [hunk 3]
```diff
@@ -74,9 +74,10 @@ for providing values to variables using the Coder CLI:
...
    to specify variable values inline as key-value pairs.
-3. _Variables file selection_: Alternatively, you can use a variables file
+4. _Variables file selection_: Alternatively, you can use a variables file
```
Comment (EdwardAngert) [inline docs/admin/templates/extending-templates/variables.md:80]:
```suggestion
1. _Variables file selection_: Alternatively, you can use a variables file
```

**Review by EdwardAngert — CHANGES_REQUESTED** [review]

**Comment by phorcys420** [comment]
@EdwardAngert out of curiosity, why make everything `1.`?

**Comment by EdwardAngert** [comment]
> @EdwardAngert out of curiosity, why make everything `1.`?

@phorcys420 it's mostly preference - markdown doesn't really care what numbers you use as long as the spacing is correct. I default to using the [Google developer style guide](https://developers.google.com/style/) and the GitLab docs style guide](https://docs.gitlab.com/ee/development/documentation/styleguide/). My personal preference is to use `1.` so that in cases like this, when we add or remove a number, it's all already `1.` so you don't have to adjust other steps to match.

- [GitLab docs style guide](https://docs.gitlab.com/ee/development/documentation/styleguide/#choose-between-an-ordered-or-unordered-list)
- [MD029](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md029---ordered-list-item-prefix)

**Review by EdwardAngert — APPROVED** [review]

**Comment by EdwardAngert** [comment]
pushing a commit to see if I can fix those errors and get this merged

**Pull request merged**